src/schema/annotation.yaml

```diff
@@ -4,25 +4,36 @@ title: Annotation Module for NMDC Schema
 description: >-
   This module in the schema is for representing annotations including functional annotations of proteins and other gene products,
   as well as controlled terms for describing things like metabolites
+default_range: string
 
 see_also:
   - https://github.com/microbiomedata/nmdc-metadata/issues/176
 
 default_curi_maps:
-  - obo_context
   - idot_context
+  - obo_context
 
 prefixes:
-  nmdc: https://microbiomedata/meta/
+  COG: http://example.com/
```

Comment:
if we're going to make this change we should use the correct URLs

I think the linter will suggest the right URLs, cc @pkalita-lbl

Reply:
The `canonical_prefixes` rule is a pretty light-weight wrapper around https://github.com/linkml/prefixmaps. You can provide the `prefixmaps_contexts` option as a list of strings that will be passed to prefixmaps' `load_multi_context` to prioritize some prefix registries over others (the prefixmaps docs can explain it better than me 😁). But that's all there is in terms of configuration for now. What else would be helpful for you?